README.md

```diff
@@ -1 +1,23 @@
-# SeedAvatarStudio
\ No newline at end of file
+# SeedAvatarStudio
+
+
```

Comment:
Please add a summary sentence to describe the project itself. E.g., consider using the following sentence:

> A real-time motion capture utility to control pre-defined 3D avatars and export the generated animations to videos or live streaming sources.